love/src/components/ObservatorySummary/ObservatorySummary.jsx

```diff
@@ -1,56 +1,94 @@
 import styles from './ObservatorySummary.module.css';
 import SummaryPanel from '../GeneralPurpose/SummaryPanel/SummaryPanel';
 import Label from '../GeneralPurpose/SummaryPanel/Label';
 import Value from '../GeneralPurpose/SummaryPanel/Value';
 import Title from '../GeneralPurpose/SummaryPanel/Title';
 
+const POOLING_TIME = 3000;
+
 export default class ObservatorySummary extends Component {
-  static propTypes = {};
+  static propTypes = {
+    /** Simonyi Telescope Observing Mode */
+    simonyiObservingMode: PropTypes.string,
+    /** Simonyi Telescope Tracking State */
+    simonyiTrackingState: PropTypes.string,
+    /** Auxiliary Telescope Observing Mode */
+    auxtelObservingMode: PropTypes.string,
+    /** Auxiliary Telescope Tracking State */
+    auxtelTrackingState: PropTypes.string,
+    /** Control Location */
+    controlLocation: PropTypes.object,
+    /** Last Updated Control Location info */
+    lastUpdated: PropTypes.instanceOf(Date),
+  };
+
+  static defaultProps = {
+    simonyiObservingMode: 'Unknown',
+    simonyiTrackingState: 'Unknown',
+    auxtelObservingMode: 'Unknown',
+    auxtelTrackingState: 'Unknown',
+    controlLocation: null,
+    lastUpdated: null,
+  };
 
-  static defaultProps = {};
+  componentDidMount() {
+    this.props.subscribeToStream();
+  }
+
+  componentWillUnmount() {
+    this.props.unsubscribeToStream();
+  }
 
   render() {
+    const {
+      simonyiObservingMode,
+      simonyiTrackingState,
+      auxtelObservingMode,
+      auxtelTrackingState,
+      controlLocation,
+      lastUpdated,
+    } = this.props;
+
+    const controlLocationName = controlLocation
+      ? controlLocation.name.charAt(0).toUpperCase() + controlLocation.name.slice(1)
+      : 'Unknown';
+
     return (
       <div className={styles.container}>
         <SummaryPanel className={styles.row2}>
           <Title wide>Simonyi Telescope</Title>
-          <Label>Summary</Label>
-          <Value>
-            <StatusText status={'invalid'}>UNKNOWN</StatusText>
-          </Value>
           <Label>Operation Mode</Label>
-          <Value>UNKNOWN</Value>
-          <Label>Obsv. Mode</Label>
-          <Value>{this.props.simonyiObservingMode}</Value>
+          <Value>Unknown</Value>
+          <Label>Observation Mode</Label>
+          <Value>{simonyiObservingMode}</Value>
           <Label>Tracking Mode</Label>
-          <Value>UNKNOWN</Value>
+          <Value>{simonyiTrackingState}</Value>
           <Label>Power Source</Label>
-          <Value>UNKNOWN</Value>
+          <Value>Unknown</Value>
         </SummaryPanel>
 
         <SummaryPanel>
           <Title wide>Vera C. Rubin Observatory</Title>
           <Label>Control</Label>
-          <Value>UNKNOWN</Value>
+          <Value>
+            <span title={`Last updated: ${lastUpdated?.toUTCString()}`}>{controlLocationName}</span>
+          </Value>
           <Label>Power Source</Label>
-          <Value>UNKNOWN</Value>
+          <Value>Unknown</Value>
```

Comment:
This "Unknown" value is correct?

Reply:
Same as the previous comment, we still don't have this source of information.